web/rest/brand/features/provider/matchList/putMatchList.feature

```diff
@@ -24,3 +24,25 @@ Feature: Update match lists
           "id": 3
       }
     """
+
+  Scenario: Match list updates ignores company values
+    Given I add Brand Authorization header
+    When I add "Content-Type" header equal to "application/json"
+    And I add "Accept" header equal to "application/json"
+    And I send a "PUT" request to "/match_lists/3" with body:
+    """
+      {
+          "name": "updatedMatchlist",
+          "company": 1
+      }
+    """
+    Then the response status code should be 200
+    And the response should be in JSON
+    And the header "Content-Type" should be equal to "application/json; charset=utf-8"
+    And the JSON should be like:
+    """
+     {
+          "name": "updatedMatchlist",
+          "id": 3
```

Comment:
Shouldn't you check company here?

Reply:
I think I can't actually test this through rest API because that field is not part of the spec :-\

Reply:
I think I can't actually test this through rest API because that field is not part of the spec :-\

Reply:
Test removed.

The value of company is not provided in the response, so it can not be checked. The matchlist could be retrieved again and assume the value has not changed because GET access_control still works (it filters out entries with companyId filled),  but don't like it too much either.